pages/linux/lastlog.md

```diff
@@ -0,0 +1,19 @@
+# lastlog
+
+> Show the most recent login of all users or of a given user.
+
+- Display the most recent login of all users:
+
+`lastlog`
+
+- Display lastlog record of the specified user:
+
+`lastlog -u {{username}}`
+
+- Display only lastlog records before than 7 days:
```

Comment:
Perhaps `Display only records older than 7 days` reads better. No need to mention "lastlog" again.

Reply:
Yes, but I think `before` can be hint of the arg `-b`